src/app/views/fireside/_sidebar/AppFiresideSidebarFiresideSettings.vue

```diff
@@ -256,6 +303,28 @@ function onClickChatMods() {
 
 							<AppSpacer vertical :scale="6" />
 
+							<AppFormGroup
+								name="targettables"
+								class="sans-margin-bottom"
+								:label="$gettext(`Tagged to`)"
+								small
+							>
+								<AppPostTargets
+									:communities="selectedCommunities"
+									:max-communities="1"
+									:realms="selectedRealms"
+									:max-realms="maxRealms"
+									:targetable-communities="targetableCommunities"
+									can-add-realm
+									can-remove-realms
+									:with-community-channels="false"
+									@remove-realm="removeRealm"
+									@select-realm="attachRealm"
+								/>
```

Comment:
Targets have the same background as our sidebar, making things blend together too much. Can we have `AppPostTargets` tag a `bg-color` prop?

Reply:
what would this prop accept? is it a name for a css variable? do we have scaffolding classes for applying these colors or what?

Reply:
sorry, this ended up being very annoying to do, had to propogate this through 3 components all the way to pill, and pill-bi and sometimes it could just be an <a> tag and it none of this was even used by this PR.

instead i just pass in a `bg-color-offset` bool prop, and if its given it applies a class that does a deep selector on `.pill` to change its background color. its very limited yes but its local to the only component that uses it rn, so im fine with it.

when we have more usecases for this we can do this